python/dask_cudf/dask_cudf/__init__.py

```diff
@@ -1,29 +1,69 @@
-# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
 
+import dask.dataframe as dd
+from dask import config
 from dask.dataframe import from_delayed
 
 import cudf
 
 from . import backends
 from ._version import __git_commit__, __version__
-from .core import DataFrame, Series, concat, from_cudf, from_dask_dataframe
-from .groupby import groupby_agg
-from .io import read_csv, read_json, read_orc, read_text, to_orc
```

Comment:
Top-level imports must pass through `dask.dataframe` in order for the `"query-planning"` to be picked up correctly. This should not break/change "legacy" behavior.